mandatory/3-playing-computer.js

```diff
@@ -7,28 +7,42 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  A1: because b was not defined.
   2. Remove the line that throws the error.
+  A2: console.log(b) was deleted
   3. What is printed to the console?
+  A3: 
+2
+6
+4
+9
+6
+13
+8
   4. How many times is "f1" called?
+  A4: 2 times
   5. How many times is "f2" called?
+  A5: 3 times
   6. What value does the "a" parameter take in the first "f1" call?
+  A6:6
```

Comment:
I'm afraid this one isn't quite correct, it's asking what the value of `a` within the f1 function the first time it is executed.

```javascript
const f1 = function (a, b) { // <--- The value of a here, which is not the same as `a` outside the function
  return a + b + "f1";
};
```

A good way of checking this is to use `console.log()` within the f1 function.
```javascript
console.log(a, ' <--- f1 a value');
```